app/src/main/res/values/strings.xml

```diff
@@ -369,12 +369,12 @@
     <string name="orbot_install_dialog_negative">Dismiss</string>
     <string name="orbot_install_dialog_neutral">website</string>
     <string name="orbot_website_link">https://guardianproject.info/apps/org.torproject.android/</string>
-    <string name="orbot_download_link_website">https://github.com/guardianproject/orbot/releases</string>
-    <string name="orbot_install_activity_error">Something went wrong, try after sometime</string>
+    <string name="orbot_download_link_website" translatable="false">https://github.com/guardianproject/orbot/releases</string>
+    <string name="orbot_install_activity_error">Something went wrong. Try after later.</string>
 
-    <string name="settings_orbot_heading">Setup Orbot</string>
+    <string name="settings_orbot_heading">Set up Orbot</string>
```

Comment:
re: see discussion below on why these texts are call-to-action (`verbs`).

Reply:
I left this here, because it acually sets something up, rather than turn it on (with some of the others).
The icons aren't shown, but rather downloaded first. And implicitly a toggle for "Icons in __" explains what it does.